docs/plugins/plugin-visible-range.md

```diff
@@ -51,9 +52,19 @@ Visible longitude as two angles.
 
 Visible latitude as two angles.
 
+#### `usePanoData`
+- type: `boolean`
+- default: `false`
+
+Use cropped panorama data as visible range.
+
 
 ## Methods
 
 #### `setLatitudeRange(range)` | `setLongitudeRange(range)`
 
 Change or remove the ranges.
+
```

Comment:
those have no interest at being in the documentation, actually I think the methods should be private

Reply:
I thought they might be useful if someone wants to update the visible range at a later point. Then they could do `setLatitudeRange(getPanoLatitudeRange())`.

But I'm fine with making them private.